tools/lua_style_and_check.sh

```diff
@@ -0,0 +1,100 @@
+#!/bin/bash
+# Copyright (c) 2016 Ford Motor Company,
+# All rights reserved.
+#
+# Redistribution and use in source and binary forms, with or without
+# modification, are permitted provided that the following conditions are met:
+#
+# * Redistributions of source code must retain the above copyright notice, this
+#   list of conditions and the following disclaimer.
+#
+# * Redistributions in binary form must reproduce the above copyright notice,
+#   this list of conditions and the following disclaimer in the documentation
+#   and/or other materials provided with the distribution.
+#
+# * Neither the name of Ford Motor Company nor the names of its
+#   contributors may be used to endorse or promote products derived from
+#   this software without specific prior written permission.
+#
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+# AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+# IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+# DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE
+# FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+# DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+# SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+# CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+# OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+# OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+  
+TEXT_DEFAULT="\\033[0;39m"
+TEXT_INFO="\\033[1;32m"
+TEXT_ERROR="\\033[1;31m"
+TEXT_UNDERLINE="\\0033[4m"
+TEXT_BOLD="\\0033[1m"
+
+GIT_DIFF_CHECK_LIST="/test_scripts/Policies" 
+
+
+##################################################################
+### Check for odd whitespace
+##################################################################
+
+echo -e "$TEXT_INFO" "Checking odd whitespaces" "$TEXT_DEFAULT"
+git diff --check --cached --color -- .$GIT_DIFF_CHECK_LIST | cat
+if [ "$?" -ne "0" ]; then
+  echo -e "$TEXT_ERROR" "Your changes introduce whitespace errors"
+  echo -e " Aborting commit." "$TEXT_DEFAULT"
+
+  exit 1
+fi
+echo -e "$TEXT_INFO" "PASSED" "$TEXT_DEFAULT"
+
+##################################################################
```

Comment:
@LuxoftAKutsan comments were simplified in c7bb3d62f7daa8fbe83ad799c9467954cd7e0912